gumdrop_derive/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "gumdrop_derive"
-version = "0.4.0"
```

Comment:
I prefer to do changes to the project's version as a standalone commit. Revert this line and I'll do a release commit, then publish after merging.

Reply:
Sure :)